.github/workflows/deployment.yml

```
@@ -0,0 +1,62 @@
+# Copied from https://github.com/notlmn/browser-extension-template/blob/master/.github/workflows/deployment.yml
+name: Deployment
+
+on:
+  workflow_dispatch:
+  # You can manually trigger a deployment on GitHub.com
+  # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+  schedule:
+    - cron: '31 13 * * 3'
+
+jobs:
+  Build:
+    outputs:
+      created: ${{ steps.daily-version.outputs.created }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: npm install
+      - run: npm test # This includes the build
+      - uses: fregante/daily-version-action@v1
+        name: Create tag if necessary
+        id: daily-version
+      - uses: notlmn/release-with-changelog@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          template: '{commits}'
+          exclude: '^Meta|^Document|^Readme|^Lint'
```

Is it worth adding '^Meta|^Document|^Readme|^Lint|dependencies$' as an implied value for `true`?
```suggestion
          exclude: true
```

This way it's still not the default, but matches the common usage

---

Well let's re-consider the default options for `notlmn/release-with-changelog` to use the options we have for RGH and other extensions. PR welcome.

---

Moved to https://github.com/notlmn/release-with-changelog/issues/20